test/e2e/ginkgo_wrapper/describe.go

```diff
@@ -0,0 +1,78 @@
+package ginkgo_wrapper
```

Comment:
Since it's going to be massive we should be careful with the name here are some options:

```golang
ovnframework "github.com/ovn-org/ovn-kubernetes/test/e2e/framework"

ovnframework.Describe(...
```


```golang
ginkgonode "github.com/ovn-org/ovn-kubernetes/test/e2e/ginkgonode"

ginkgonode.Describe(...
```

Another alternative is instead of chaging the Nodes add something new that wraps the text 

Describe(OVN(....)) <- this OVN function is new

Reply:
i didnt want to name it framework because i didnt want to name it overly generic when its only going to be a ginkgo wrapper.

Happy to change it if maintainer doesn't but I dont favor any of your suggestions.